docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2.1'
 
 services:
     crossdock:
-        image: crossdock/crossdock
+        image: crossdock/crossdock:filter-crossdock
```

Comment:
Nice! Obv lets get the Crossdock PR in first so that we can land this pinned to master 👍